source/frontend/src/features/contacts/styles.ts

```diff
@@ -25,6 +25,10 @@ export const HalfWidthLayout = styled.div`
   height: 100%;
   width: 50%;
   min-width: 93rem;
+  @media only screen and (max-width: 1199px) {
```

Comment:
fluidly adjust the view down to a minimum, where scrolling on the x axis is allowed.